clang/unittests/Interpreter/InterpreterTest.cpp

```diff
@@ -114,6 +114,13 @@ TEST_F(InterpreterTest, Errors) {
 
   RecoverErr = Interp->Parse("var1 = 424;");
   EXPECT_TRUE(!!RecoverErr);
+
+  Err = Interp->Parse("int x = 5; auto capture = [&]() { return x * 2; };").takeError();
```

Comment:
Can we move this test perhaps somewhere in `test/Interpreter/lambda.cpp`?

Reply:
Hmmm @vgvassilev 

Don't you think we should look at this from an error recovery perspective rather than something for lambdas ? If that's the case, the test is placed correctly.

The above test cause clang-repl to crash but should work if wrapped in a function or not presented as a top level decl.

Reply:
There is no such a distinction in the test suite.

Reply:
I meant I see quite some error recovery stuff being added in this place hence I added it here !